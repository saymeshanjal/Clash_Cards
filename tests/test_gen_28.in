99
12 57 2
16 220 1
4 106 3
12 379 1
6 96 5
16 297 1
4 336 4
4 165 4
4 206 4
10 333 3
4 42 5
10 65 2
10 291 4
14 248 3
12 66 4
16 17 2
8 346 5
16 364 1
10 293 2
10 26 2
2 187 4
12 311 1
16 203 4
8 177 2
6 318 5
6 134 4
8 1 1
10 377 5
8 293 5
2 329 1
6 137 5
14 126 4
10 117 4
10 216 2
12 355 2
16 229 3
8 136 5
4 12 4
8 376 4
6 280 5
10 315 2
2 169 5
10 309 5
10 130 1
12 279 3
10 374 4
6 33 5
4 354 5
4 149 1
14 16 3
10 380 3
8 180 5
2 81 4
8 333 1
16 170 2
12 358 4
4 184 2
10 1 2
10 186 3
10 136 1
12 132 3
2 359 1
12 96 2
2 163 2
10 26 2
2 296 5
6 49 2
12 221 3
8 282 5
16 350 3
6 163 1
6 84 5
14 185 3
8 25 2
12 390 5
4 54 1
8 0 5
14 5 4
8 281 1
6 176 3
4 154 2
12 4 3
4 2 3
10 181 1
8 378 1
12 363 2
14 143 2
4 253 4
4 296 5
8 10 3
8 284 3
16 97 2
16 162 2
4 175 3
10 250 4
12 367 2
10 70 5
8 159 1
16 8 4
