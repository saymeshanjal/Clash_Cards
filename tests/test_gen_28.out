286 287 351
6435 6435 0
1 5 0
462 462 0
1 19 0
6435 6435 0
0 5 1
0 5 1
0 5 1
110 142 0
0 6 0
76 77 99
44 133 75
1652 1780 0
208 441 275
539 2303 10028
8 62 0
6435 6435 0
76 77 99
36 54 162
0 2 0
462 462 0
3809 5406 3655
20 21 29
1 19 0
1 14 5
35 35 0
45 207 0
8 62 0
1 1 0
1 19 0
910 1520 1002
44 133 75
76 77 99
286 287 351
6307 6563 0
8 62 0
0 5 1
8 42 20
1 19 0
76 77 99
0 2 0
45 207 0
126 126 0
430 494 0
44 133 75
1 19 0
0 6 0
3 3 0
459 2973 0
110 142 0
8 62 0
0 2 0
35 35 0
4081 4082 4707
208 441 275
1 2 3
0 126 126
110 142 0
126 126 0
430 494 0
1 1 0
286 287 351
0 1 1
36 54 162
0 2 0
5 6 9
430 494 0
8 62 0
6307 6563 0
10 10 0
1 19 0
1652 1780 0
20 21 29
219 705 0
3 3 0
0 70 0
0 3312 120
35 35 0
6 14 0
1 2 3
28 896 0
0 6 0
126 126 0
35 35 0
286 287 351
1078 1079 1275
0 5 1
0 6 0
3 67 0
27 43 0
4081 4082 4707
4081 4082 4707
1 5 0
44 133 75
286 287 351
45 207 0
35 35 0
1 11673 1196
